src/components/partner/partner.repository.ts

```diff
@@ -106,8 +106,8 @@ export class PartnerRepository extends DtoRepository(Partner) {
           sub
             .with('projList')
             .raw('UNWIND projList as project')
-            // TODO: have this match from line 92 so we don't have to match this twice
-            .match([
+            // TODO: I don't think this is needed.... more thought on this later.
+            .optionalMatch([
```

Comment:
Shouldn't `matchProjectScopedRoles` & L103 be moved into this subquery after deduping projects matched?

Reply:
I don't think it should. if `matchProjecScopedRoles` andL103 were moved into this subquery, we would have to double-collect the scoped roles in order to collect all scopedRoles of all projects that member is connected to. 

.... fixed the match with partnership. Fixing org now.

Reply:
org fixed